submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import json
+
+from helpers import FileManager, Vocabulary, EpithetGenerator
+
+happy_path = '../../resources/data.json'
+sad_path = '../../resources/foo.bar'
+csv_path = '../../resources/data.csv'
+fake_path = '../../fake.json'
+with open(happy_path, 'r') as f:
+    data = json.loads(f.read())
+
+
+class TestFileMangager(unittest.TestCase):
+
+    def test_get_extension(self):
+        self.assertEqual(
+            FileManager.get_extension(happy_path), 'json'
+        )
+        self.assertEqual(
+            FileManager.get_extension(csv_path), 'csv'
+        )
+        self.assertEqual(
+            FileManager.get_extension('test/testo/westo/'), ''
+        )
+        with self.assertRaises(TypeError):
+            FileManager.get_extension(1)
+
+    def test_read_json(self):
+
+        self.assertEqual(
+            FileManager.read_json(happy_path), data
+        )
+
+        with self.assertRaises(FileNotFoundError):
+            FileManager.read_json(sad_path)
+
+
+class TestVocabulary(unittest.TestCase):
+
+    def test_from_file(self):
+        self.assertEqual(
```

Comment:
Each assertion should be its own method so that the test runner can report which test is failing.